GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -10,14 +10,20 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
     override public func viewDidLoad() {
         super.viewDidLoad()
-        let items = Item.getItems()
-        configureDataSource(items: items)
-        toggleListDisplay(items: items)
+        dataSource = ItemCollectionViewDataSource()
```

Comment:
I would move this line into `configureDataSource` since it is conceptually part of doing that.

Reply:
makes sense! implemented change